.gitignore

```diff
@@ -27,7 +27,6 @@ dependency-reduced-pom.xml
 *.diff
 *.patch
 *.tmp
-/distribution/bin/
```

Comment:
为什么删除它？
Why delete it?

Reply:
Because i add `seata-namingserver.bat`  in `/distribution/bin/` , the file will be ignored unless i delete this line in .gitignore.

Reply:
> Because i add `seata-namingserver.bat` in `/distribution/bin/` , the file will be ignored unless i delete this line in .gitignore.

你可以仅在本地修改，而不是把.gitignore提交上来